### ISSUE TITLE
Pricing Toggle Button

### ISSUE THREAD

**Issue by baudoinalkali** [baudoinalkali/chattr#59]
A few things:

Please remove the blue outline when it's in focus.

Would it be time intensive to have the ability to drag it as well as click it?

**Comment by ogdenstudios** [comment]
We can remove the blue outline, but it's a violation of the ADA and WCAG guidelines to have no focus state at all. Do you have an option for us to indicate focus?

I can add a touch handler quickly. A few hours, tops. Although we'll probably have to fiddle with it cause it'll need some configuration: how sensitive it is, how big the touch area is, etc.)

**Comment by baudoinalkali** [comment]
I mentioned the outline issue, but they seemed to not care.

Let's move forward with the touch handler.

**Comment by ogdenstudios** [comment]
Sounds good to me.
